packages/web-components-stencil/src/components/link/test/link.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { Link } from '../link';
+
+describe('lux-link', () => {
+  it('renders', async () => {
+    const page = await newSpecPage({
+      components: [Link],
+      html: `<lux-link></lux-link>`,
+    });
+    expect(page.root).toEqualHtml(`
+      <lux-link>
+        <mock:shadow-root>
+          <utrecht-link class="lux-link">
+            <slot></slot>
+          </utrecht-link>
+        </mock:shadow-root>
+      </lux-link>
+    `);
+  });
+
+  it('passes props', async () => {
+    const page = await newSpecPage({
+      components: [Link],
+      html: `<lux-link download="download.file" href="url.html" target="_parent"></lux-link>`,
```

Comment:
Wellicht kunnen we voor de prop args een variabele maken zodat het duidelijk is dat de invoer en uitvoer gelijk moeten zijn. Nu kan je het interpreteren als dat het toevallig gelijk is aan elkaar